.github/workflows/legend-release.yml

```diff
@@ -0,0 +1,373 @@
+name: Trigger Legend Stack Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      legend-shared-release-version:
+        description: "New release version for legend shared"
+        required: false
+      legend-pure-release-version:
+        description: "New release version for legend pure"
+        required: false
+      legend-engine-release-version:
+        description: "New release version for legend engine"
+        required: false
+      legend-sdlc-release-version:
+        description: "New release version for legend sdlc"
+        required: false
+      legend-depot-release-version:
+        description: "New release version for legend depot"
+        required: false
+
+env:
+  OWNER: finos
+  RELEASE_TOKEN: ${{ secrets.FINOS_GITHUB_TOKEN }}
+  EVENT_TYPE: legend-stack-release
+  WORKFLOW_ID: legend-stack-release.yml
+
+jobs:
+  release:
```

Comment:
Instead of having one 'release' job with all these steps, we could have a couple of jobs e.g. 'release-legend-shared', 'release-legend-endinge' etc. with the 3 steps per job needed for that.

This would look better in the GitHub Actions UI, and it would allow us to run certain releases in parallel (e.g. legend-shared and legend-pure).

With the 'needs' parameter we can express the dependency between our artifacts - mimicking the maven dependency tree (this will likely be even more beneficial when we split into even more projects)

Reply:
done